lease-kubernetes/src/main/scala/akka/coordination/lease/kubernetes/internal/KubernetesApiImpl.scala

```diff
@@ -238,16 +238,19 @@ PUTs must contain resourceVersions. Response:
   }
 
   private def makeRequest(request: HttpRequest, timeoutMsg: String): Future[HttpResponse] = {
-    val httpRequest =
-      if (settings.secure)
+    val response =
+      (if (settings.secure)
```

Comment:
do we need the `(`?

Reply:
Ah, sorry, left over after doing other things here. I'll remove